terraform/environments/core-network-services/route53.tf

```diff
@@ -102,6 +102,22 @@ resource "aws_route53_record" "bichard7" {
   ]
 }
 
+# MRLA/MAAT location
+resource "aws_route53_record" "mrla" {
```

Comment:
This record isn't needed, this is creating a name server record which would send all DNS queries to the ns records detailed in this record.  We don't need to do this here as we want to have control of the domain.  What we do need to do is once this hosted zone is created, get the NS records for this new zone and give them to ops engineering so that they can create a record like this one to forward queries on to us